client/src/components/cullingexclusionzones/CullingExclusionZones.tsx

```diff
@@ -0,0 +1,47 @@
+import { UnculledZone } from "../../api/liberationApi";
+import { selectUnculledZones } from "../../api/unculledZonesSlice";
+import { useAppSelector } from "../../app/hooks";
+import { LayerGroup, LayersControl, Circle } from "react-leaflet";
+
+interface CullingExclusionCirclesProps {
+  zones: UnculledZone[];
+}
+
+const CullingExclusionCircles = (props: CullingExclusionCirclesProps) => {
+  return (
+    <>
+      <LayerGroup>
+        {props.zones.map((zone, idx) => {
+          return (
+            <Circle
+              key={idx}
+              center={zone.position}
+              radius={zone.radius}
+              color="#b4ff8c"
+              fill={false}
+              interactive={false}
+            />
+          );
+        })}
+      </LayerGroup>
+    </>
+  );
+};
+
+export default function CullingExclusionZones() {
+  const data = useAppSelector(selectUnculledZones).zones;
```

Comment:
This assures that updates that arrive in the slice, get pushed through, using `useListUnculledZonesQuery` like before doesn't do what I expected. If I had to guess, it queries the zones from the front-end instead of the back-end.

Reply:
`useListUnculledZonesQuery` does query the backend, but there's automatic caching to reduce unnecessary calls. The auto-generator needs to be told how to invalidate the cache though, and that probably wasn't set up for this query yet.

Ditching it and pushing the data into redux is totally fine with me though. It's often the easier approach IME.